### PR TITLE
Add Node.js v8 migration docs

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: articles/migrations/extensibility-node8.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+| couchbase | ~1.2.1 | ~2.3.5 |
+| jsonwebtoken | ~0.4.1^* | ~7.4.1 (w/ compatibility shim) |
+| knex | ~0.6.3 | ~0.13.0 |
+| mongo-getdb | ~1.2.0^* | ^2.2.0 |
+| mongodb | ~1.3.15^* | ^2.2.0 |
+| mysql | 2.0.0-alpha8^* | ^2.0.0 |
+| node-cassandra-cql | ^0.4.4 | ^0.5.0 |
+| request | ~2.27.0 | ~2.81.0 |
+| pg | ^4.3.0^* | ^4.5.7 |
+| bcrypt | ~0.8.5^* | ~0.8.7 |
```
Comment (fyockm) [inline articles/migrations/extensibility-node8.md]: Might have one more module to add here (xml2json). Will confirm tomorrow.

File: articles/migrations/extensibility-node8.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+When using the **/connections** endpoints, Custom Database Scripts can be retrieved or updated using **options.customScripts**
+
+Similarly, you can find Custom Social Connections in **options.scripts.fetchUserProfile**.
+
+You will need to manually copy over any Hooks-related code that you use.
+
+### Migration assistance
+
+We have created a migration assistant to help ease the copying of code between production and development tenants.
```
Comment (fyockm) [inline articles/migrations/extensibility-node8.md]: Link to migration assistant, once @FadyMak copies into `auth0` org.

Comment (ghost) [reply articles/migrations/extensibility-node8.md]: @fyockm you can use this URL (it won't change): https://github.com/auth0/webtask-migration-assistant

I created the repo and will merge there when complete

File: articles/migrations/extensibility-node8.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+## Summary of the migration
+
+The Webtask runtime powering the following Auth0 features utilize Node.js v4:
+
+* Rules
+* Hooks
+* Custom database connections
+* Custom social connections
+
+Due to the end of long-term support (LTS) for Node.jsv.4, we will be migrating the Webtask runtime to use Node.js v8.
```
Comment (ghost) [inline articles/migrations/extensibility-node8.md]: `Node.jsv.4` -> `Node.js v4`

File: articles/guides/extensibility-node8.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+If you have manually pinned modules, you may need to manually update them so that you code runs with Node.js v8.
+
+For example, you must change
+
+`var mysql = require(‘mysql@2.0.0-alpha8’);`
+
+to
+
+`var mysql = require(‘mysql@2.0.0’);`
+
```
Comment (fyockm) [inline articles/guides/extensibility-node8.md]: or more simply
`var mysql = require(‘mysql’);`

**Comment by khorne3** [comment]
@fyockm I've incorporated the fixes you've requested so far.

File: articles/migrations/extensibility-node8.md (hunk 1 above)
Comment (fyockm) [reply articles/migrations/extensibility-node8.md]: @khorne3 please add
```
| xml2json | ~0.10.0^* | ~0.11.2 |
```

**Pull request merged**